Description: revert inconsistent ls quoting
 This is a revert of upstream commit 109b9220cead6e979d22d16327c4d9f8350431cc.
 Info changed to reflect current upstream intentions.

Bug-Debian: https://bugs.debian.org/813164

Index: coreutils-8.26/NEWS
===================================================================
--- coreutils-8.26.orig/NEWS	2016-11-30 13:11:57.000000000 -0500
+++ coreutils-8.26/NEWS	2016-12-09 09:09:56.000000000 -0500
@@ -217,9 +217,6 @@
   df now prefers sources towards the root of a device when
   eliding duplicate bind mounted entries.
 
-  ls now quotes file names unambiguously and appropriate for use in a shell,
-  when outputting to a terminal.
-
   join, sort, uniq with --zero-terminated, now treat '\n' as a field delimiter.
 
 ** Improvements
Index: coreutils-8.26/doc/coreutils.texi
===================================================================
--- coreutils-8.26.orig/doc/coreutils.texi	2016-11-24 06:08:31.000000000 -0500
+++ coreutils-8.26/doc/coreutils.texi	2016-12-09 09:09:56.000000000 -0500
@@ -7831,8 +7831,8 @@
 
 You can specify the default value of the @option{--quoting-style} option
 with the environment variable @env{QUOTING_STYLE}@.  If that environment
-variable is not set, the default value is @samp{shell-escape} when the
-output is a terminal, and @samp{literal} otherwise.
+variable is not set, the default value is @samp{literal}, but this
+default may change to @samp{shell-escape} in a future version of this package.
 
 @item --show-control-chars
 @opindex --show-control-chars
Index: coreutils-8.26/src/ls.c
===================================================================
--- coreutils-8.26.orig/src/ls.c	2016-11-22 15:04:32.000000000 -0500
+++ coreutils-8.26/src/ls.c	2016-12-09 09:09:56.000000000 -0500
@@ -1678,7 +1678,6 @@
       if (isatty (STDOUT_FILENO))
         {
           format = many_per_line;
-          set_quoting_style (NULL, shell_escape_quoting_style);
           /* See description of qmark_funny_chars, above.  */
           qmark_funny_chars = true;
         }
